org.panorama-research.mobstr.tim/model/mobstr-tim.xcore

```diff
@@ -244,4 +245,26 @@ class RefinesItemDefinition extends DirectedTraceLink {
  class RefersToAnalysisResult extends DirectedTraceLink {
  	refers Element [1] source // Element -- Safety Case
  	refers Element [1..*] target // Element -- Failure in FTA
- }
\ No newline at end of file
+ }
+ 
+ /*******************************
+  *       OFFIS Extensions
+  *******************************/
```

Comment:
Is this comment meaningful to outsiders down the line?

Reply:
I removed that comment